Modules/CaPTkCommon/cmdapps/CaPTkCohortJsonFromFilesystemCmdApp.cpp

```diff
@@ -0,0 +1,101 @@
+#include <mitkCommandLineParser.h>
+
+#include <CaPTkCohortJsonFromFilesystem.h>
+#include <CaPTkWriteJsonToFile.h>
+
+#include <json/json.h>
```

Comment:
we are not linking to json but utilizing Qt's json module.

Reply:
That will be done in the future, and maybe by alex